library/julia

```diff
@@ -1,5 +1,6 @@
 # maintainer: InfoSiftr <github@infosiftr.com> (@infosiftr)
 
+0.4: git://github.com/docker-library/julia@433ee54173ff0b8900045f5a81cad96b34558e9f
```

Comment:
Cool!  Just give us a prod when you've got it updated (or let me know if you'd rather I just roll it into #1123). :+1: